packages/cmf/src/bootstrap.md

```diff
@@ -0,0 +1,40 @@
+# react-cmf bootstrap
+
+This api help you to start without importing much than the api
+
+Quick start:
+
+```javascript
+import { api } from '@talend/react-cmf';
+import components from './components';
+import sagas from './sagas';
+
+configure.initialize();
+const app = api.bootstrap({
+    components,
+	settingsURL: '/settings.json',
+	saga: sagas.appSaga, // function* appSaga() {...}
+	sagaRouterConfig: sagas.routes, // { '/foo/bar': function* bar() {...}, ... },
+    ...manyMoreOptions
+});
+app.render();
+```
+
+## Options
+
+| name | type | default | description |
+| -- | -- | -- | -- |
+| settingsURL* | string | undefined | **REQUIRED** this URL to fetch the cmf settings.json file |
+| appId | string | 'app' | the default html element id to get to render react |
+| history | Object | `{ hashHistory } from 'react-router'` | the history object to control the router |
```

Comment:
If I'm not wrong it can take only one of the history from react-router.
It would be better to precise this list of possible values : `browserHistory`, `hashHistory`, `createMemoryHistory()` from `react-router`

Reply:
Yes with react-router v3. But in v4 it s a little bit different. You have to import it from 'history'
https://reacttraining.com/react-router/web/api/Router/history-object